views/js/test/runner/plugins/probes/latency/test.js

```diff
@@ -129,10 +129,12 @@ define([
             getProxy: function() {
                 return {
                     sendVariables: function(trace) {
-                        var traceData = _.indexBy(probeData, function(entry) {
-                            return (entry.marker ? entry.marker + '-' : '') + entry.type + '-' + entry.id;
+                        return new Promise(function() {
```

Comment:
What is the purpose of this promise? There is no resolver, nor rejecter. It will be endless.
At least, please add a timeout and make the promise resolve at some point.

Reply:
```
testRunner.getProxy().sendVariables(traceData, true).catch
```
shows us an error, that result of sendVariables doesn't have a `catch` method

Reply:
fixed